pkg/apis/networking/v1beta1/doc.go

```diff
@@ -19,4 +19,4 @@
 // +groupName=networking.antrea.tanzu.vmware.com
 
 // Package v1beta1 is the v1beta1 version of the Antrea NetworkPolicy API messages.
-package v1beta1
+package v1beta1 // import "github.com/vmware-tanzu/antrea/pkg/apis/networking/v1beta1"
```

Comment:
what does the extra comment mean?

Reply:
Sorry, I tried to generate code before and this line is legacy code. Removed.